packages/roosterjs-content-model-plugins/lib/imageEdit/utils/getContentModelImage.ts

```diff
@@ -0,0 +1,14 @@
+import { getSelectedSegments } from 'roosterjs-content-model-dom';
+import type { ContentModelImage, IEditor } from 'roosterjs-content-model-types';
+
+/**
+ * @internal
+ */
+export function getContentModelImage(editor: IEditor): ContentModelImage | null {
```

Comment:
where is this used? This will give you a cloned object but not the original image object. Is that ok?

Reply:
This is used to get the image dataset